tests/Shell/Commands/CommandTest.php

```diff
@@ -67,18 +67,18 @@ public function test_validate_path_command()
 
     public function test_serialize_no_args_no_opts()
     {
-        $cmd = new Command('phpunit');
+        $cmd = new Command('test');
```

Comment:
What test failure was being caused by using an actual executable?

Reply:
if you dont have phpunit on your machine, test fails